### PR TITLE
integrate with delve

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <keao.yang@yahoo.com>

### What problem does this PR solve?

Integrate `delve` inside the pods to help developers debug.

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:

### Checklist

Tests

- [x] Manual test

Run according to the README.md in `images/chaos-dlv`

### CODE REVIEW

File: images/chaos-dlv/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+FROM golang:1.15.11 AS build-env
+
+ENV GO111MODULE on
+
+WORKDIR /
+RUN wget https://github.com/go-delve/delve/archive/refs/tags/v1.6.1.tar.gz
+RUN tar xvf v1.6.1.tar.gz
+WORKDIR /delve-1.6.1
+RUN go install github.com/go-delve/delve/cmd/dlv
```
Comment (STRRL) [inline images/chaos-dlv/Dockerfile:11]: how about:

``` dockerfile
FROM golang:1.16
ENV GO111MODULE on
RUN go install github.com/go-delve/delve/cmd/dlv@v1.6.1
```

rest LGTM

Comment (YangKeao) [reply images/chaos-dlv/Dockerfile]: Good! Modified.

**Review by STRRL — APPROVED** [review]
LGTM!

**Review by Hexilee — APPROVED** [review]
LGTM

**Pull request merged**